pkg/web/routes.go

```diff
@@ -102,18 +102,24 @@ func (s *Server) setupRoutes() (err error) {
 	s.router.GET("/transactions/:id/info", authenticate, s.TransactionsInfo)
 	s.router.GET("/send-envelope", authenticate, s.SendEnvelopeForm)
 	s.router.GET("/send-message", authenticate, s.SendMessageForm)
-	s.router.GET("/verify", s.VerifySunriseUser)
-	s.router.GET("/sunrise-accept", s.SunriseMessagePreview)
```

Comment:
These routes will be changed to `/sunrise/verify` and `/sunrise/accept` is that ok?

Reply:
That's fine. Should we change `send-message` to `sunrise/message` to make it a little more clear that it's related to that feature? I wasn't sure what would be preferred when initially creating the routes.

Reply:
Ah, I didn't realize `send-message` was a sunrise route; yes we should probably change it so that it's enabled/disabled by the middleware. I propose `sunrise/send` is that ok?

Reply:
Er, or I guess `POST sunrise/send` and `GET sunrise/message`?

Reply:
That'll work!